Internship2022TeamPatri/Internship2022TeamPatri/Networking/Services/AuthApiManager.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  AuthApiManager.swift
+//  Internship2022TeamPatri
+//
+//  Created by Vlad Silviu Hagiu on 02.08.2022.
+//
+
+import Foundation
+import FirebaseAuth
+import UIKit
+import FirebaseFirestore
+
+class AuthApiManager {
+    static let sharedInstance = AuthApiManager()
+    private let db = Firestore.firestore()
+    
+    func loginAPI(email: String?, password: String?, completion:  @escaping (Bool, Error?) -> Void) {
```

Comment:
The completion can be just with the optional error. you don't need Bool here. The bool flag should be a public variable in the AuthApiManager, so you can access it later in the app.